apps/docs/src/guide/essentials/index.md

```diff
@@ -1,3 +1,5 @@
 # Essentials
 
-This guide will cover the essentials required for interacting with programs built using Sway.
\ No newline at end of file
+This guide will cover the essentials required for interacting with the Fuel Network.
+
+This includes building and modifying transactions and interacting with Sway program types; contracts, scripts and predicates.
```

Comment:
```suggestion
This includes building and modifying transactions and interacting with Sway program types: contracts, scripts and predicates.
```

: instead of ;.

Reply:
One day I'll be able to use a semi colon correctly